Tests/test_PDB_PSEA.py

```diff
@@ -0,0 +1,116 @@
+# Copyright 2022 by Valentin Vareskic (valentin.vareskic@gmail.com).
+# All rights reserved. This code is part of the Biopython distribution
+# and governed by its license.  Please see the LICENSE file that should
+# have been included as part of this package.
+
+"""Tests for PDB PSEA."""
+
+import unittest
+from Bio.PDB.PSEA import run_psea, psea, psea2HEC, PSEA
+from Bio.PDB import PDBParser
+from subprocess import getoutput
+from Bio import MissingExternalDependencyError
+import os
+
+os.environ["LANG"] = "C"
+
+if "command not found" or "'psea' is not recognized" in getoutput("psea -h"):
```

Comment:
Actually, this conditional will always evaluate to `True` because it is interpreted as `if ("command not found") or ("'psea' is not recognized" in getoutput("psea -h"))` and the first string (command not found) evaluates to True.

Reply:
Good catch - I didn't read that closely enough.

Reply:
First I was like where is the problem here, then it came to me like oooh 😄